sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -62,7 +62,7 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
    * Returns the current default database in this session.
    */
   override def currentDatabase: String =
-    sparkSession.sessionState.catalogManager.currentNamespace.toSeq.quoted
+    sparkSession.sessionState.catalogManager.currentNamespace.quoted
```

Comment:
hmm does this make a difference?

Reply:
oh I see. It's so tricky to have implicit class for both `Seq[String]` and `Array[String]`. We should clean it up in the future.

Reply:
>  It's so tricky to have implicit class for both Seq[String] and Array[String]

Yea 👀